### ISSUE TITLE
Can you update the Shift+I recommendation in the ReadMe?

### ISSUE THREAD

**Issue by ScottChesworth** [Timtam/AccessiChords#14]
Setting AccessiChords up on a new machine just now, I noticed that Shift+I is the recommended shortcut for insertion in your ReadMe. At the time of writing Shift+I and I were duplicated on the OSARA key map, but since then, we've mapped those keys to two separate insertion actions with different behaviours of the edit cursor (I doesn't advance the edit cursor, Shift+I does).
Can you change your recommended shortcut? Alt+I is available, and seems to go nicely with the other recommendations.

**Comment by Timtam** [comment]
Yeah, can deffinitely do that, I wanted to rework the entire keymap too, but it seems like the other keystrokes are used now as well, and I didn't find a viable key layout yet that would work for all actions.
